pkg/operator/ceph/config/conditions.go

```diff
@@ -84,6 +87,24 @@ func setCondition(context *clusterd.Context, namespace, name string, newConditio
 	}
 }
 
+// translatePhasetoState convert the Phases to corresponding State
+func translatePhasetoState(phase cephv1.ConditionType) string {
```

Comment:
Please add a comment to this method that explains why the translation is necessary, for example that changes to the state enum are considered breaking changes. Otherwise it will be difficult to remember the purpose of this change.